Sources/BraintreeVenmo/Public/BraintreeVenmo/BTVenmoDriver.h

```diff
@@ -67,6 +67,7 @@ typedef NS_ENUM(NSInteger, BTVenmoDriverErrorType) {
     On success, you will receive an instance of `BTVenmoAccountNonce`; on failure, an error; on user
     cancellation, you will receive `nil` for both parameters.
 */
+// We should rework this to use a `BTVenmoRequest` object and consolidate to a single authorization method. There are projects coming to move to Payment Context and add more customization to the experience
```

Comment:
This is a great idea! 

Addressed in PR https://github.com/braintree/braintree_ios/pull/602